.github/workflows/python-integration-test.yml

```diff
@@ -57,18 +57,25 @@ jobs:
       - name: Set up QEMU
         uses: docker/setup-qemu-action@v2
 
-      - name: Free up disk space
-        run: |
-          dev/ci/util_free_space.sh
-
       - name: Python Client Integration Test
         id: integrationTest
+        env:
+          EXTERNAL_START_GRAVITINO: true
```

Comment:
Better change to “START_EXTERNAL_GRAVITINO”

Reply:
DONE.